edx_event_bus_kafka/internal/consumer.py

```diff
@@ -146,11 +146,12 @@ def reset_offsets(consumer, partitions):
             if offset_timestamp is None:
                 return
 
-            # Get the offset from the epoch in seconds.
-            offset_timestamp_s = int(offset_timestamp.timestamp())
+            # Get the offset from the epoch (Kafka expects offsets in milliseconds for offsets_for_times, see
+            # https://kafka-python.readthedocs.io/en/master/apidoc/KafkaConsumer.html#kafka.KafkaConsumer.offsets_for_times)
```

Comment:
Maybe something along these lines? Anyway, I'll go ahead and approve, and you can decide what makes the most sense docs-wise.

```suggestion
            # Get the offset from the epoch. Kafka expects offsets in milliseconds for offsets_for_times. Although
            # this is undocumented in the libraries we're using (confluent-kafka and librdkafa), for reference
            # see the docs for kafka-python:
            # https://kafka-python.readthedocs.io/en/master/apidoc/KafkaConsumer.html#kafka.KafkaConsumer.offsets_for_times
```